CMakeLists.txt

```
@@ -16,6 +16,14 @@ ament_target_dependencies(${PROJECT_NAME} "rclcpp" "rclcpp_action")
 target_link_libraries(${PROJECT_NAME} ${CMAKE_THREAD_LIBS_INIT})
 target_include_directories(${PROJECT_NAME} PUBLIC include)
 
+# A library to detect a realtime kernel and set thread priority, if one is found
+add_library(thread_priority STATIC src/thread_priority.cpp)
+target_include_directories(thread_priority PUBLIC
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+  "$<INSTALL_INTERFACE:include>"
+)
```

why not simply:

```suggestion
target_include_directories(thread_priority PUBLIC include)
```

or will this not work for some reason? I am using this all the time and didn't have any issue until now.

---

Not planning to change this. See the discussion above.